Podfile

```diff
@@ -50,7 +50,7 @@ def wordpress_ui
 end
 
 def wordpress_kit
-  pod 'WordPressKit', '~> 8.7', '>= 8.7.1'
+  pod 'WordPressKit', '~> 8.8.0'
```

Comment:
Hey @mokagio can you confirm the `WordPressKit` pod version?

Reply:
I read this question in two ways:

- Is 8.8.0 the right version for this PR?
- Is 8.8.0 the right version before the upcoming code freeze?

**8.8.0 is the correct version for this PR.**

This can be verified by comparing `trunk` with `8.8.0`: it won't show the PR with the changes we need here, meaning that it's part of `8.8.0`.

Another way I usually check this is to see if the PR shows the tag in GitHub, e.g.:

![image](https://github.com/wordpress-mobile/WordPress-iOS/assets/1218433/c19c3856-413c-4c18-b8aa-a864fb58a72c)

But for some reason, this particular PR doesn't even though [the code is in the releases](https://github.com/wordpress-mobile/WordPressKit-iOS/compare/8.7.1...8.8.0):

![image](https://github.com/wordpress-mobile/WordPress-iOS/assets/1218433/da00b850-3fac-4dbb-92be-1aa29f38c20e)

Maybe a GitHub caching hiccup?

However, **8.8.0 is not the right version for the upcoming release.** 8.9.0 is. Unfortunately, I run into some hiccups with the deployment, so that's not ready yet. Meaning that for all effect and purpose this is the right version to use.

:shipit: 
---

One more thing. I recommend using `~> 8.8` so that we can automatically install compatible updates.